src/openrct2/ride/Vehicle.cpp

```diff
@@ -7243,7 +7243,7 @@ void Vehicle::UpdateAdditionalAnimation()
                             }();
                             int32_t directionIndex = sprite_direction >> 1;
                             auto offset = SteamParticleOffsets[typeIndex][directionIndex];
-                            steam_particle_create(x + offset.x, y + offset.y, z + offset.z);
+                            steam_particle_create(CoordsXYZ{x + offset.x, y + offset.y, z + offset.z});
```

Comment:
```suggestion
                            steam_particle_create(CoordsXYZ{x, y, z} + offset});
```

Reply:
The type of offset is not CoordsXYZ nor a type that there is a + operator available.